src/pages/documentation/platform-assurance/tutorials/validation-summary.md

```diff
@@ -1,22 +1,18 @@
-# Validation Summary
+# Validation Editor
 
 ## Overview
 
-You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in an Adobe Experience Platform Assurance session. Each function receives the events in an Assurance session. You can write functions to validate your client configuration, event conditions, tests and use cases.
+You may use the Validation Editor feature to quickly and easily manage JavaScript functions to validate events in an Adobe Experience Platform Assurance session. Each function receives the events in an Assurance session. You can write functions to validate your client configuration, event conditions, tests and use cases.
 
 ## Getting Started
 
 To use this view, complete the following steps:
 
-1. [Set up Adobe Experience Platform Assurance](../set-up.md)
-2. [Create](../index.md#creating-sessions) and [connect](../index.md#connecting-to-a-session) to a Assurance session
```

Comment:
What was the reason for this change, the old links seem to be working - https://aep-sdks.gitbook.io/docs/adobe-assurance/project-griffon/using-project-griffon/validation-summary-and-project-griffon#getting-started